examples/fastapi/requirements.txt

```diff
@@ -1,8 +1,6 @@
-moesifasgi
+moesifasgi==0.2.0
```

Comment:
I'd suggest don't specify any version. Since it's an example, we'd always want to use the latest version of moesifasgi